Store/src/database/database.cs

```diff
@@ -130,30 +111,114 @@ PRIMARY KEY (id)
 
     public static async Task LoadPlayer(CCSPlayerController player)
     {
-        using MySqlConnection connection = Connect();
-
-        dynamic result = await connection.QueryFirstOrDefaultAsync(@"SELECT * FROM store_players WHERE SteamID = @SteamID",
-        new
+        if (player == null || !player.IsValid || player.IsBot)
         {
-            player.SteamID
-        });
+            return;
+        }
 
-        Server.NextFrame(() =>
+        Credits.SetOriginal(player, -1);
+        Credits.Set(player, -1);
+
+        try
         {
-            if (result == null)
+            using (MySqlConnection connection = Connect())
             {
-                Instance.GlobalStorePlayers.Add(new Store_Player
+
+                // Perform three related queries at once: player information, item information, and equipment information.

```

Comment:
Due to poor real-time performance, when you buy a new item

Reply:
Additionally, querying for each individual player every time should be better than querying all players when loading configurations, because the performance is likely to be poor when you have more than 500,000 players.

Reply:
You can also add items and modify scores for players in real-time from an external database; they just need to rejoin the server to see the changes.

Reply:
@e54385991 
I'd be glad if you contact me in discord. Thanks for pr :)